macros/core/src/snapshots/sails_macros_core__program__tests__gprogram_generates_handle_for_multiple_services_with_empty_and_non_empty_routes.snap

```diff
@@ -11,6 +11,27 @@ impl MyProgram {
         MyService
     }
 }
+impl sails_idl_meta::ProgramMeta for MyProgram {
```

Comment:
are those autogenerated? maybe exclude it from source contrrol?

Reply:
They are autogenerated at the very first run. If one is happy with the output, they accept them to be a reference. For the subsequent runs, if output differs, a test fails and a diff is shown.
https://crates.io/crates/insta